### ISSUE TITLE
Cant get Dictionary value by key in View 

### ISSUE THREAD

**Issue by AMBULATUR** [riganti/dotvvm#1047]
At ViewModel
ex. to reproduce 
```
  Dictionary<int, int> ProfileRoles {get;set;} = new Dictionary<int, int>() {
                    //ProfileID,RoleID
                    {1,1},
                    {2,0}
                };  

                int SelectedProfileID {get;set;} = 2
```

ex. at View: 

```
<div class="Physical" IncludeInPage="{value: ProfileRoles[SelectedProfileID] == 1}">
```

![](https://i.imgur.com/sTgI8Xo.png)
![](https://i.imgur.com/vadTB62.png)

**Comment by acizmarik** [comment]
What version of DotVVM do you use? 

Support for dictionaries is already implemented and will be released with version 3.1 (until then you can try a preview version that should already support the example you provided). Dictionaries were not supported in previous versions. However, we are aware of the fact that some constructs might have worked by accident in older versions.

**Comment by AMBULATUR** [comment]
Yep, on 3.1 works fine, problem was on 3.0.4.
